source/includes/steps-create-system-dsn-linux.yaml

```diff
@@ -0,0 +1,68 @@
+title: Check the location of your ``.so`` files
+ref: check-location
+level: 4
+content: |
+  The |odbc-driver-name| files are named ``libmdbodbca.so`` and
+  ``libmdbodbcw.so``. After downloading the |odbc-driver-name|,
+  move them to your ``/usr/local/lib`` directory.
```

Comment:
I'm not sure: Do we need a '/' after lib to better indicate a directory?

Reply:
I don't think so. The fact that we refer to it as a directory makes it clear enough.

Reply:
Ack.